src/versioning-strategies/dependency-manifest.ts

```diff
@@ -23,9 +23,12 @@ import {
   PatchVersionUpdate,
 } from '../versioning-strategy';
 
-const DEPENDENCY_UPDATE_REGEX =
+const RENOVATE_DEPENDENCY_UPDATE_REGEX =
```

Comment:
Can you please rename this back to what it was before? This is a generic dep update message, rather than specific only to Renovate.

Reply:
done!